internal/app/cf-terraforming/cmd/util.go

```diff
@@ -99,15 +99,6 @@ func sharedPreRun(cmd *cobra.Command, args []string) {
 
 	var options []cloudflare.Option
 
-	if accountID != "" {
-		log.WithFields(logrus.Fields{
-			"account_id": accountID,
-		}).Debug("configuring Cloudflare API with account")
-
-		// Organization ID was passed, use it to configure the API
-		options = append(options, cloudflare.UsingAccount(accountID))
```

Comment:
this is going in 4.x of the provider however, no endpoints use this global setting so it can go.